docs/mlops-crash-course/tong-ket/tom-tat-noi-dung.md

```diff
@@ -4,11 +4,11 @@
 
 ## Tóm tắt nội dung khóa học
 
-Chúng ta đã cùng nhau đi qua một chặng đường dài, từ bước làm rõ bài toán kinh doanh, thực hiện POC chứng minh hiệu quả của model, tiếp tới là xây dựng các data pipeline, training pipeline, sau đó đóng gói để serve model, và cuối cùng là tự động hóa tất cả các bước trên. Trong suốt cả quãng đường này, chúng ta cũng làm quen với rất nhiều tool, đi kèm với best practices được đúc rút và thu thập từ nhiều nguồn tài liệu khác nhau có thể kể tới như Airflow, MLFLow, Feast, .v.v. Việc học cách sủ dụng các tool này là cần thiết, song không nên quá phụ thuộc hay lạm dụng nó mà quên đi vấn đề chính là làm thế nào để giải quyết bài toán một cách đơn giản và hiệu quả nhất.
+Chúng ta đã cùng nhau đi qua một chặng đường dài, từ lúc làm rõ bài toán kinh doanh, thực hiện POC chứng minh hiệu quả của model, tiếp tới là xây dựng các data pipeline, training pipeline, sau đó đóng gói để serve model và cuối cùng là tự động hóa tất cả các bước trên. Trong suốt cả quãng đường này, đã cùng nhau làm quen với rất nhiều tool, đi kèm với best practices được đúc kết và thu thập từ nhiều nguồn tài liệu khác nhau có thể kể tới như Airflow, MLFLow, Feast, .v.v. Việc học cách sử dụng các tool này là cần thiết, song không nên quá phụ thuộc hay lạm dụng nó mà quên đi vấn đề chính là làm thế nào để giải quyết bài toán một cách đơn giản và hiệu quả nhất.
 
-Chúng ta đồng thời đã tìm hiểu về kiểm thử trong ML, nó không đơn thuần chỉ là kiểm thử code như bên software, mà chúng ta còn phải quan tâm tới data và model nữa. Việc kiểm thử này cũng quan trọng như là xây dựng model vậy, vì nếu không có kiểm thử thì đầu ra sẽ không thể tin cậy được.
+Đồng thời chúng ta đã tìm hiểu về kiểm thử trong một hệ thống ML, nó không đơn thuần chỉ là kiểm thử code như bên software, mà chúng ta còn phải quan tâm tới data và model nữa. Việc làm này quan trọng như là xây dựng model vậy, vì nếu không có kiểm thử thì đầu ra sẽ không thể tin cậy được (garbage in - garbage out).
 
-Feature store cũng là một thành phần thú vị trong chuỗi bài giảng của chúng ta. Nó đang xuất hiện ở ngày càng nhiều công ty, giúp quản lý, đánh giá và chia sẻ feature một cách dễ dàng giữa các thành viên trong team, và giữa các team trong toàn tổ chức. Việc ứng dụng feature store sẽ giảm thiểu rất nhiều công sức của bạn, bên cạnh đó cũng vô cùng tiềm năng trong việc cải thiện chất lượng model thông qua việc nâng cao chất lượng feature.
+Feature store cũng là một thành phần thú vị trong chuỗi bài giảng của chúng ta. Nó đang xuất hiện ngày càng nhiều ở các công ty. Giúp cho việc quản lý, cũng như đánh giá chia sẻ feature một cách dễ dàng giữa các thành viên trong team và giữa các team trong toàn tổ chức. Ứng dụng này sẽ giảm thiểu rất nhiều công sức của bạn, bên cạnh đó việc làm này vô cùng tiềm năng trong lúc cải thiện chất lượng model thông qua việc nâng cao chất lượng feature.
```

Comment:
`Giúp cho việc quản lý, cũng như đánh giá chia sẻ feature một cách dễ dàng giữa các thành viên trong team và giữa các team trong toàn tổ chức` ko phải 1 câu hoàn chỉnh

Reply:
em sửa lại rồi ạ